packages/runtime-tools-process-dev-ui-webapp/src/components/containers/ProcessFormContainer/ProcessFormContainer.tsx

```diff
@@ -57,7 +57,9 @@ const ProcessFormContainer: React.FC<ProcessFormContainerProps & OUIAProps> = ({
               onSubmitSuccess(id);
             })
             .catch((error) => {
-              const message = error.response ? error.response.data : error.message;
+              const message = error.response
+                ? ` ${error.message}   ' ${error.response.request.statusText} '`
```

Comment:
Nip picking here. In this case I'd like to have a separator between both messages. I'd use `error.response.statusText` instead too.

Something like:


```suggestion
                ? `${error.response.request.statusText} ': ${error.message} `
```

Notice I removed some blank spacess from the string too.

Reply:
Thank you for the comment and suggestion. It's done. Please have a look now.